### PR TITLE
'Distribute points' questions: increase the width of status info message #7876

### LINKED ISSUE #7876: 'Distribute points' questions: increase the width of status info message
Current:
Very narrow width. Appears squished.

image

Suggested:
Increase width so that it appears as one line.

### PR DESCRIPTION
**Pull request by uxxSam**
Fixes #7876

I removed the offset and made the width to be 100% so that the info message appears in one line. 
Here are how the messages look before:
For sm:
distribute-pt-sm
For md:
distribute-pt-md
And here are how they look after:
For sm:
after-distribute-pt-sm
For md:
after-distribute-pt-md

### CODE REVIEW

**Comment by uxxSam** [comment]
Travis CI build failed on NoSuchElementException; working on this right now

File: src/main/resources/feedbackQuestionConstSumSubmissionFormTemplate.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div>
     ${constSumSubmissionFormOptionFragments}
     <div id="constSumInfo-${questionIndex}-${responseIndex}" class="form-group" ${constSumOptionVisibility}>
-        <div class="col-sm-offset-4 col-sm-6 col-md-offset-2 col-md-3">
+        <div class="col-sm-12 col-md-12">
```
Comment (whipermr5) [inline src/main/resources/feedbackQuestionConstSumSubmissionFormTemplate.html]: There's no need for `col-md-12` when `col-sm-12` is already set.

Comment (uxxSam) [inline src/main/resources/feedbackQuestionConstSumSubmissionFormTemplate.html]: That's right! I changed all to col-sm-12 :D

**Comment by uxxSam** [comment]
Ready for review!

**Pull request merged**